frontend/src/components/cluster-dashboard/cluster-graph-view/ClusterGraphView.tsx

```diff
@@ -0,0 +1,233 @@
+    }
+  };
+
+  const loadGraphData = (): void => {
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const frameGraphData = (clusterData: Cluster): any => {
+      const leader = {
+        shadow: {
+          enabled: true,
+          color: styles.leaderNodeColor,
+          size: 15,
+          x: 1,
+          y: 1
+        }
+      };
+      const oldest = {
+        borderWidth: 3,
+        borderWidthSelected: 0,
+        color: {
+          border: styles.oldestNodeColor,
+          highlight: { border: styles.oldestNodeColor },
+          hover: { border: styles.oldestNodeColor }
+        },
+        shapeProperties: { borderDashes: [10, 10] }
+      };
+      //eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const graph: any = {
+        nodes: [
+          {
+            id: 0,
+            label: "Akka Cluster",
+            image: clusterUrl,
+            size: 40,
+            borderWidth: 1,
+            borderWidthSelected: 2,
+            color: {
+              border: styles.primaryColor,
+              background: styles.secondaryColor,
+              highlight: {
+                border: styles.primaryColor,
+                background: styles.secondaryColorLighter
+              },
+              hover: {
+                border: styles.primaryColor,
+                background: styles.secondaryColorLighter
+              }
+            }
+          }
+        ],
+        edges: []
+      };
+
+      if (!clusterData || !clusterData.members) {
+        return graph;
+      }
+
+      clusterData.members.forEach((member, index) => {
+        const memberTitle = (
+          <React.Fragment>
+            <div>
+              <span className="font-bold disp-inline-blk">node : </span>
+              <span
+                className={
+                  "status disp-inline-blk " + member.status.toLowerCase()
+                }
+              ></span>
+              <span className="disp-inline-blk">{member.node}</span>
+            </div>
+            <div>
+              <span className="font-bold">nodeUid : </span>
+              <span>{member.nodeUid}</span>
+            </div>
+            <div>
+              <span className="font-bold">status : </span>
+              <span>{member.status.toLowerCase()}</span>
+            </div>
+            <div>
+              <span className="font-bold">roles : </span>
+              <span>{member.roles.join(", ")}</span>
+            </div>
+            <div>
+              {member.node === clusterData.oldest && (
+                <span className="type oldest disp-inline-blk">Oldest</span>
+              )}
+              {member.node === clusterData.leader && (
+                <span className="type leader disp-inline-blk">Leader</span>
+              )}
+            </div>
+          </React.Fragment>
+        );
+
+        const memberConfig = {
+          id: index + 1,
+          label: "<b>o </b>" + member.node.split("://")[1],
```

Comment:
Is `o` the intended direction for this label? Perhaps using an html entity would be nicer as a status indicator if the css isn't accessible for a `::before` pseudoelement.
```suggestion
          label: "<b>· </b>" + member.node.split("://")[1],
```
or
```suggestion
          label: "<b>&middot; </b>" + member.node.split("://")[1],
```

But really a CSS solution would be best, with a simple `::before`

```
.statusLabel.good::before {
  content: '';
  width: 1rem;
  height: 1rem;
  vertical-align: middle;
  border-radius: .5rem;
  background-color: green;
  margin-right: .5rem;
}
```

Reply:
Unfortunately vis network graphs don't let you add icons or unicode characters to node labels. Although it is a hack, this seems like the best option for now. I have to go back and figure out if there's another way to solve this.